src/pages/admin-settings.tsx

```diff
@@ -0,0 +1,319 @@
+import React, { useState } from 'react';
+import { BsPersonFill, BsThreeDotsVertical } from 'react-icons/bs';
+import { data, taskQueuesData } from '../data/data';
+
+const AdminEditUser = ({ setShowModal }: { setShowModal: React.Dispatch<React.SetStateAction<boolean>> }) => {
```

Comment:
Nit: I'd make a separate component called AdminEditUser.tsx and then import that component into this file. That way, it's easier to read the code & keeps our files smaller.